servers/physics_2d/godot_physics_server_2d.cpp

```diff
@@ -1044,40 +1044,67 @@ void GodotPhysicsServer2D::joint_clear(RID p_joint) {
 
 void GodotPhysicsServer2D::joint_set_param(RID p_joint, JointParam p_param, real_t p_value) {
 	GodotJoint2D *joint = joint_owner.get_or_null(p_joint);
-	ERR_FAIL_COND(!joint);
+	ERR_FAIL_COND(joint);
```

Comment:
```suggestion
	ERR_FAIL_COND(!joint);
```
You forgot the `!` when restoring